.github/workflows/build.yml

```diff
@@ -68,6 +61,14 @@ jobs:
         run: |
           make build
 
+      - name: Running source code format check
+        run: |
+          make check-fmt
+
+      - name: Running static analysis
+        run: |
+          make lint-c
+
```

Comment:
Why do we need to move lint and check-fmt targets?

Reply:
`check-fmt` and `lint-c` require a meson build (or at least the setup) due to `ninja`. Meson uses ninja as a backend by default and generates all targets on setup.

Reply:
Right, I missed that, thanks